fh-ios-sdk/Sync/FHSyncDataset.m

```diff
@@ -383,7 +384,7 @@ - (void)startSyncTask:(NSDictionary *)info {
         syncLoopParams[@"fn"] = @"sync";
         syncLoopParams[@"dataset_id"] = self.datasetId;
         syncLoopParams[@"query_params"] = self.queryParams;
-        syncLoopParams[@"meta_data"] = self.metaData;
+        syncLoopParams[@"meta_data"] = self.syncMetaData;
```

Comment:
I believe this should be the customMetaData instead. This meta_data is sent to the cloud. The syncMetaData is only used locally and do not need to be sent to the cloud.